### DISCUSSION TITLE
Captain

### DISCUSSION THREAD

**Discussion by TomekPL-1** [secretcowlevel/DoomtrooperPublic#5] [Card Balance]
Captain stats could be lowered

**Comment by framerate** [comment]
Suggestion on stats?

**Comment by Kite80** [comment]
Captain is a top tier warrior for Capitol, the only one for now; other corporations have venusian marshal, eradicator deathdroid, inquisitor majoris and an entire circus of beasts for DL; I'd rather add a similar warrior to those factions which still don't have one, like Mishima which can have shadow walkers and Imperials with their air cavalry

**Reply by TomekPL-1** [reply]
Currently his stats are higher that Big Bob and Mith Hunter
 F: 6 S: 8 A: 7 V: 9
I would suggest
 F: 5 S: 7 A: 6 V: 7
